utils/utils.go

```diff
@@ -138,6 +138,52 @@ func HandlePanic() {
 	}
 }
 
+// GenerateGithubActions generates a GitHub Actions workflow file for Keploy
+func GenerateGithubActions(logger *zap.Logger, path string, appCmd string) {
+	// Determine the path based on the alias "keploy"
+	logger.Info("Determining the path of the keploy binary based on the alias \"keploy\"")
+
+	keployPath := "/usr/local/bin/keploy" // Default path
```

Comment:
the default path is /usr/local/bin/keploybin not keploy

Reply:
Refactored